### ISSUE TITLE
dev_mode

### ISSUE THREAD

**Issue by kajdzo** [reduxframework/redux-framework#2843]
Hello, 
I'm using redux to create theme options panel and I have issue when I get it from 
dev_mode => true to dev_mode=>false all my options just disappear and in console I get following error: 
Uncaught TypeError: $.cookie is not a function  in redux.js. I would appreciate any suggestions. 
Thanks

**Comment by kprovance** [comment]
We could begin with the information we need from you:  https://github.com/reduxframework/redux-framework/blob/master/CONTRIBUTING.md

**Comment by kprovance** [comment]
Closing due to lack of reply for required information.  If further help is needed, please view the link provided and post back the necessary data.

**Comment by kajdzo** [comment]
Hello again, 
Sorry for not responding right away, I haven't come to office yesterday. I'm developer that is using redux to develop a theme. I have last version of redux. I also created several custom field types, using boilerplate you guys provided(thanks for that), and included those in redux. I did tried to remove one at the time so I would discover if there is a problem but that didn't showed any results.
